java/piranha/src/test/resources/com/uber/piranha/XPMethodChainCasesDoNotAllowMethodChain.java

```diff
@@ -64,8 +64,8 @@ public static void foobar(Parameter cp) {
     System.out.println("done!");
     // Matches API
     // Do not match API
-    cp.put("", "other_flag", true);
-    cp.put("", "other_flag", false);
+    cp.put(sp.otherFlag(), true);
```

Comment:
I don't follow the reason for this change. Should this have gone in a separate PR?

Reply:
Actually I updated the test case. Rather than passing the flag as an argument, it should actually pass the corresponding abstract method.

Reply:
Supporting this scenario is the reason for this PR.